src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -1558,7 +1558,7 @@ public void RunIdleOperations()
         {
             foreach (var index in _indexes)
             {
-                index.Cleanup();
+                index.Cleanup(everything: true);
```

Comment:
Not sure that this should be here, that runs on a timer, no?

Reply:
This runs on a timer, but only if the database is idle

Reply:
In will happen every 10min (forced), which means that this will clear the searcher every 10min? Not sure if we want this.

Reply:
I also have some doubts about it

Reply:
`RunIdleOperations` is called only if the database is idle, and in that case we have no issue to clean this, I think.
Or do I miss something?

Reply:
Check the second condition here:
https://github.com/ravendb/ravendb/blob/b860e8ee4bb8a4a1070fdd9b5f094eccd7a3095b/src/Raven.Server/ServerWide/ServerStore.cs#L2380